package.json

```diff
@@ -89,6 +89,7 @@
     "lib/"
   ],
   "scripts": {
+    "debug-test": "npm test -- --timeout 0",
```

Comment:
We should set the timeout in the specific test `this.timeout(0)`

Reply:
this doesn't really need to get committed. it's a QoL thing because I am always hit by the timeout exception whenever I debug tests in express

Reply:
put another way, I would like to stop having to modify package json locally every time I want to debug tests

Reply:
removed